### DISCUSSION TITLE
Fix filters in projects list header

### DISCUSSION THREAD

**Discussion by kamil7x** [community/community#6585] [Projects and Issues] [answered]
![image](https://user-images.githubusercontent.com/7629808/138283969-86a8cc46-967a-4171-b29f-3879896c93c9.png)

**Problem:**
When Projects (beta) is selected, clicking on Open or Closed link, navigated to proper filter, but in old Projects view

**Expected result**
New projects list should be filtered instead

**Comment by vcaballero-salle** [comment, marked as answer]
I think this has been resolved. Can you confirm?
